src/mysql_recv.erl

```diff
@@ -75,6 +75,7 @@ start_link(Host, Port, LogFun, Parent) when is_list(Host), is_integer(Port) ->
 	spawn_link(fun () ->
 			   init(Host, Port, LogFun, Parent)
 		   end),
+	erlang:monitor(process, RecvPid),
```

Comment:
As I suggested in #37, I would move this call to mysql_conn:
```erlang
    case mysql_recv:start_link(Host, Port, LogFun, self()) of
	{ok, RecvPid, Sock} ->
	    erlang:monitor(process, RecvPid), % <-- here
	    case mysql_init(Sock, RecvPid, User, Password, LogFun, FoundRows) of
```

Other than that suggestion, your change looks good to me.